Rogue-Ape-Studios-XR/Secrets-of-Ignacios
# 🍱 Feature/water vfx

## Content

Added a splash effect, and a water shader for the projectiles

## Changes

### Added
`Assets/VFX/Water/Water Splash.vfx` : Was created. Water Visual effect
`Assets/Materials/Water/Water.shadergraph` : Was created. Water shader.



## Testing

The feature / Bugfix can be testing by following these steps:

1. Drag the new water shader onto an object (preferably a sphere). It should look like water.
2. Drag the water splash vfx into the scene and play it. It should create a splash effect that goes along with rotation

![SOIwaterVFX](https://github.com/user-attachments/assets/71529e7c-09b0-413f-a779-f0098f7dcbee)
